### ISSUE TITLE
Bug report: function crash and burn with  "['features'] not found in axis"

### ISSUE THREAD

**Issue by lubianat** [jvfe/reconciler#6]
Hey @jvfe,

I was trying to split and reconcile [this SPARQL query result](https://w.wiki/igW).

77-item dataframe with names. I run:

`reconciled = reconcile(new_df["itemLabel"], type_id="Q16521")`

It splits in 7 chunks and runs. I get this when one of the frames fails: 
![image](https://user-images.githubusercontent.com/7917951/97110624-f67b4780-16b8-11eb-9477-f6b1649c8df1.png)
KeyError: "['features'] not found in axis"

A quick fix would be to add a try-except block in:

```    for column in tqdm(frames, position=0, leave=True):
        input_keys, response = return_reconciled_raw(
            column,
            type_id,
            has_property,
            reconciliation_endpoint,
        )
        parsed = parse_raw_results(input_keys, response)

        dfs.append(parsed.drop(["features"], axis=1))

```

More details, see [this notebook](https://github.com/lubianat/depicts/blob/master/bird.ipynb)

Idea: Replicate error with small example and add as test

**Comment by lubianat** [comment]
Here is my workaround, might help to understand the issue: https://github.com/lubianat/depicts/blob/master/bird_roots.ipynb

**Comment by jvfe** [comment]
Oh good catch, thank you! This is something I overlooked when I implemented the progress bar. 

I won't have the time to fix this in the next days, but if you feel confident enough with your solution, feel free to submit a pull request for it. I think the try, except block, though not elegant, would be a simple fix.

Thanks again.

**Comment by jvfe** [comment]
740dd562 implements what I believe would be a fix for this issue + a test inspired by your example. Hopefully this solves it but any further issues feel free to reopen this.

**Comment by gitonthescene** [comment]
I had this exact same issue with #11.  I posted what I think is a fix in PR #12.
